webapp/src/tutorial.tsx

```diff
@@ -433,12 +434,19 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
         }
     }
 
+    private handleResize() {
+        const options = this.props.parent.state.tutorialOptions;
+        this.setShowSeeMore(options.autoexpandStep);
```

Comment:
should probably throttle or at least use requestAnimationFrame()

Reply:
good point good point I added a 500ms debounce time - do you think that's enough?